core/src/main/java/app/majime/core/user/User.java

```diff
@@ -16,34 +16,30 @@
 public class User {
 
     @Id
-    @GeneratedValue(strategy = GenerationType.AUTO)
+    @SequenceGenerator(name="user_seq", sequenceName="user_id_seq", allocationSize = 1)
```

Comment:
```suggestion
    @SequenceGenerator(name="user_seq", sequenceName="lab_user_id_seq", allocationSize = 1)
```
Should be _lab_user_id_seq_ in _sequenceName_. The database will create with that name if we will use query from vertabelo (bcs table name)